src/main/java/sirius/biz/tenants/mongo/MongoUserAccount.java

```diff
@@ -47,15 +48,10 @@
     @Nullable
     private static MongoTenants tenants;
 
-    public static final Mapping USER_ACCOUNT_DATA = Mapping.named("userAccountData");
```

Comment:
Why is this mapping removed? This is actually used quite a lot in OX

Reply:
UserAccount.USER_ACCOUNT_DATA is already present - this is a c&p error